readme.md

```diff
@@ -1,14 +1,47 @@
-# frosty
+# `frosty`
```

Comment:
Do away with the `.
Also, there's no newline here, but there is one after other headers.